### PR TITLE
29 improve workers ux

### LINKED ISSUE #29: Workers UX improvement
When someone runs the game for the first time, all three of the available workers are mining iron. It's not immediately clear what to do to reallocate them for mining other resources, and also this gives some initial skew in favour of iron. Probably it would be more fair to begin with all workers idle

### PR DESCRIPTION
**Pull request by queleok**
* start 0 player with all workers idle
* start 1 player with all workers distributed between the resources
* denote the possibility to decrease/increase the number of workers allocated to the specific resource with angle brackets

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -9,11 +9,18 @@ mod visualization;
...
+use enum_iterator::IntoEnumIterator;
 
 fn main() {
     let mut state = GameState::new();
     state.register_player();
     state.register_player();
 
+    for res in Resource::into_enum_iter() {
+        state.handle_action(GameAction::AllocateWorker(1, res));
+    };
```
Comment (JacobALundgren) [inline src/main.rs]: It doesn't matter here but the semicolon doesn't actually do anything. The broader point that might be useful to bring up is that cargo has a built-in formatter, which I tend to run either before each commit or before merge, and it has a few small changes to make in this file including this one :)

```
cargo fmt
```

Comment (JacobALundgren) [reply src/main.rs]: Another thing that is quite helpful (but that doesn't have anything to say about your great code here) is the somewhat standard linter [`clippy`](https://github.com/rust-lang/rust-clippy)

```
cargo clippy
```

**Review by JacobALundgren — APPROVED** [review]
Looks great, thank you very much :) Might benefit from a quick formatting run as mentioned but perfect otherwise

File: src/main.rs (hunk 1 above)
Comment (queleok) [reply src/main.rs]: Thanks for the tips! Ran the auto-formatter and made another commit

**Comment by JacobALundgren** [comment]
Closes #29

**Pull request merged**